exp/capacity-planning.md

```diff
@@ -17,18 +17,16 @@ When estimating capacity, consider the following questions to better understand
 <a name="application-resources"></a>
 ## Application resources
 
-Depending on the resources that your application requires, choose a suitable [instance type](https://discourse.ubuntu.com/t/application-manifest/24197#instance-type).
-
-The instance type specifies the resources that are available to the application:
+A default resource preset will be set for every application. A resource preset specifies the resources that are available to the application:
 
 - The number of vCPU cores
 - The amount of RAM
 - The amount of disk space
 - The number of GPU slots
 
-If none of the provided instance types fits for your application, you can also manually [configure the resources](https://discourse.ubuntu.com/t/how-to-configure-available-resources/24960) according to your requirements.
+Depending on the resources that your application requires, if the default resource preset does not suit, you can choose suitable [resources](https://discourse.ubuntu.com/t/application-manifest/24197#resources) that fit your application.
```

Comment:
Just wondering where can people refer to the default resource presets.

Reply:
Linked to the topic